pymc3/distributions/dist_math.py

```diff
@@ -461,3 +461,23 @@ def infer_shape(self, node, shapes):
 def eigh(a, UPLO='L'):
     """A copy, remove with Eigh and EighGrad when possible"""
     return Eigh(UPLO)(a)
+
+
+def i0(x):
```

Comment:
these two are actually implemented in theano now: https://github.com/Theano/Theano/blob/master/theano/scalar/basic_scipy.py